src/main/java/teammates/storage/sqlentity/Instructor.java

```diff
@@ -0,0 +1,209 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.Objects;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import teammates.ui.output.InstructorPermissionRole;
+
+/**
+ * Represents an Instructor entity.
+ */
+@Entity
+@Table(name = "Instructors")
+public class Instructor extends User {
+    @OneToOne(mappedBy = "id")
+    @Column(nullable = false)
+    private int id;
```

Comment:
Not required, this field is inherited from User class.

Reply:
👍🏻. I will remove the one in Student too. Thanks samuel!